deltachat-ios/AppDelegate.swift

```diff
@@ -25,6 +25,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     private var backgroundTask: UIBackgroundTaskIdentifier = .invalid
 
     var reachability = Reachability()!
+    var hasNetwork = false
```

Comment:
why can't we use appDelegate.reachability.connection directly? instead of caching the state on our own. at least, this seems to compile.

Reply:
I fixed that and found also another bug in the already existing code which is fixed by b6f3631 as well